tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/replication/RepeatStartBackupTest.java

```diff
@@ -156,8 +171,110 @@ public void testLoopStart() throws Exception {
 
          Assertions.assertEquals(0, errors.get());
       }
+   }
+
+   @Test
+   public void testAckManagerRepetition() throws Exception {
+
+      String queueName = "queue_" + RandomUtil.randomString();
+
+      server.getConfiguration().setMirrorAckManagerQueueAttempts(300000);
+      server.getConfiguration().setMirrorAckManagerRetryDelay(1000);
+      backupServer.getConfiguration().setMirrorAckManagerPageAttempts(300000);
+      backupServer.getConfiguration().setMirrorAckManagerRetryDelay(1000);
+
+      ExecutorService executorService = Executors.newFixedThreadPool(2);
+      runAfter(executorService::shutdownNow);
+
+      AtomicInteger errors = new AtomicInteger(0);
+      AtomicBoolean running = new AtomicBoolean(true);
+
+      runAfter(() -> running.set(false));
+      CountDownLatch latch = new CountDownLatch(1);
+      CountDownLatch backupStarted = new CountDownLatch(1);
+
+      AtomicInteger messagesSent = new AtomicInteger(0);

```

Comment:
Given no messages are sent this ends up being a bit confusing, perhaps messageAcks ?